src/main/kotlin/net/atos/zac/task/TaskService.kt

```diff
@@ -256,15 +261,13 @@ class TaskService @Inject constructor(
         task: Task,
         loggedInUser: LoggedInUser,
         reden: String?
-    ) {
-        flowableTaskService.releaseTask(task, reden).run {
-            eventingService.send(
-                SignaleringEventUtil.event(
-                    SignaleringType.Type.TAAK_OP_NAAM,
-                    this,
-                    loggedInUser
-                )
+    ) = flowableTaskService.releaseTask(task, reden).apply {
```

Comment:
```suggestion
    ) = flowableTaskService.releaseTask(task, reden).also {
```

I think this is a typical use case for `also` because sending the event is sort of a side event. But yeah, quite hard to determine when to use which Kotlin scope function for sure..